WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/BlockListItem.kt

```diff
@@ -233,8 +233,16 @@ sealed class BlockListItem(val type: Type) {
     data class LoadingItem(val loadMore: () -> Unit, val isLoading: Boolean = false) : BlockListItem(LOADING_ITEM)
 
     data class ActivityItem(val blocks: List<Block>) : BlockListItem(ACTIVITY_ITEM) {
-        data class Block(val label: String, val boxes: List<Box>)
-        enum class Box {
+        data class Block(
+            val label: String,
+            val boxes: List<Box>,
+            val contentDescription: String,
+            val activityContentDescriptions: List<String>? = null
+        )
+
+        data class Box(val boxType: BoxType, val day: Int? = null)
```

Comment:
I'm not familiar with this part of the app. However, I'm wondering whether it makes sense to introduce `day` field in the `Box` class => the field isn't being used on the UI. It's being used only when we are building the ActivityItem. We might want to consider reverting some of the changes in `ActivityItem` and moving `buildActivityItem` to `PostingActivityMapper`, wdyt?

Reply:
Hi Jirka! Thanks for this observation. It's something we can think of modifying if it makes sense. The reason I put it here is the `contentDescription` is somewhat a UI element, it's just a different kind of UI for other users so I thought it made sense to include it.

Reply:
>contentDescription is somewhat a UI element, it's just a different kind of UI for other users so I thought it made sense to include it.

I absolutely agree with this statement. Having `contentDescription` field in `Block` is ok. 
However, it seems `ActivityItem` and `Box` are UIStates and `day` is not part of their UIState = it isn't being used in the view layer. We use it only to store some data we need to use while we are building the UIState.

It's not a big deal, it just feels weird.

Reply:
Okay! I totally get what you mean now when you connect it to the concept of `UIState`. Thanks for the explanation. So are you saying I should move this logic to the `UseCase` then?

Reply:
`PostingActivityMapper.buildActivityItem` returns an instance of `ActivityItem`, we pass that `ActivityItem` into `addBlockContentDescriptions` and we create a different instance of `ActivityItem` there, which also feels a bit weird. 
I think the best place for the contentDescription logic would be in `PostingActivityMapper` =>`PostingActivityMapper.buildActivityItem` would return a complete `ActivityItem`. Wdyt?

Reply:
Resolved 50224c5